lang/java/reef-applications/reef-vortex/pom.xml

```diff
@@ -20,7 +20,7 @@ under the License.
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
-    <artifactId>reef-vortex</artifactId>
+    <artifactId>vortex</artifactId>
     <name>REEF Vortex</name>
```

Comment:
Maybe also change the name to Vortex. However, I am sure that the ASF would want to call it "Apache REEF Vortex (incubating)" for now :-)